mrc-ide/tfpbrowser
# WP4: re-factor app

- refactor app to use Shiny modules
- refactor app into an R package
- add in shinytest2 tests
- create collapsible section
- add tab for RDS files
- Use rds inputs instead of html inputs
- Link treeview graphs to options to download outputs
- add unit tests
- add GitHub actions to check linting and R CMD check
